scripts/components/BucketCollections.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import { Link } from "react-router";
+
+import BucketTabs from "./BucketTabs";
+
+
+class DataList extends Component {
+  render() {
+    const {
+      bid,
+      data,
+      listLoaded,
+    } = this.props;
+    return (
+      <table className="table table-striped table-bordered record-list">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Schema</th>
+            <th>Cache Expires</th>
+            <th>Last mod.</th>
+          </tr>
+        </thead>
+        <tbody className={!listLoaded ? "loading" : ""}>{
+          data.map((collection, index) => {
+            return (
+              <tr key={index}>
+                <td>
+                  <Link to={`/buckets/${bid}/collections/${collection.id}/edit`}>
+                  {collection.id}
+                  </Link>
+                </td>
```

Comment:
Also we may probably want a link to the list of records for that collection as well.